content/news/2021/2021-10-04-how-to-monitor-events-in-your-svelte-app.md

```diff
@@ -0,0 +1,143 @@
+
+Svelte differentiates itself from other JavaScript web frameworks by NOT including a virtual DOM. Basically, it is the antithesis to React. Direct DOM manipulation was once the norm with plain old JavaScript and jQuery, but its since been replaced by the many many frameworks out there that implement a virtual/shadow DOM. 
+
+With Svelte taking a different approach to JavaScript web frameworks, we should explore how (if at all) handling events and monitoring those events works in Svelte. Open-source ❤️ open-source, so we'll use the open-source event monitoring tool, [Exceptionless](https://exceptionless.com) alongside our Svelte app. 
+
+What you'll need: 
+
+* Node.js v12 or higher 
+* A free Exceptionless account 
+* Text editor 
+
+## Getting Started
+
+We'll get started by signing up for Exceptionless. Note, you can also run this locally by grabbing the source code and [following the instructions here](https://github.com/exceptionless/Exceptionless). When you have signed up, you'll be prompted to create a new project. As we create the new project, we'll be prompted ot select the type of project. For this choose "browser application": 
+
+![New project set up](./new_project.png)
+
+This will give you the install instructions, but don't worry, I'll provide those here as well. Next, click Manage Project and you'll want to grab your API key from the API keys tab. 
+
+Now, we can generate our Svelte project. We'll use deget so that we can get a full project scaffolding set up easily. [Read more about that here](https://svelte.dev/blog/the-easiest-way-to-get-started). 
+
+```
+npx degit sveltejs/template exceptionless-svelte
+cd exceptionless-svelte
+
+npm install
+```
+
+Now, our last step before opening up a code editor is to install the new Exceptionless JavaScript client. You can do so by running this from inside your project directory: 
+
+```
+npm i @exceptionless/browser
+```
+
+Now, you can fire up the app by running: 
+
+```
+npm run dev
+```
+
+Don't worry about how it looks, we're going to instead focus on how to capture events and errors in the app using Exceptionless. Let's dive into that!
+
+## Handling Events With Exceptionless
+
+Svelte, like other JavaScript frameworks, allows you to build apps through a collection of components. However, to keep this tutorial simple, we will keep everything in out `App.svelte` file. So, let's start there. Open your `exceptionless-svelte` project in your favorite code editor, then find the `App.svelte` file. 
+
+You'll notice that with Svelte, we have a normal-looking script tag at the top of the file and then some HTML. This is because there is no virtual DOM. We're really just dealing with templating, reactive state, and plain JavaScript—which is pretty fun. 
+
+Let's import Exceptionless at the top of our script tag, and then let's start Exceptionless up. 
+
+```js
+import { Exceptionless } from "@exceptionless/browser"
+export let name;
```

Comment:
I'd say leave the `export let name;` line out to keep focused on what you are trying to show.

Reply:
Good call!